### DISCUSSION TITLE
Stack layers on top of each other

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#867] [Q&A] [answered]
Hi,

Is it possible to stack render layers on top of each other? I tried this:
Layer 1: Mapsforge layer
Layer 2: OpenStreetMap bitmap tile layer with opacity 0.5

But only the top layer (Layer 2) gets rendered.

**Comment by demiantres** [comment]
I just noticed that there is an example for this in the MapilionMvtActivity example. For some reason the stacking does not work properly when I add the second layer to a GroupLayer.

**Comment by devemux86** [comment, marked as answer]
See the [MapilionMvtActivity](https://github.com/mapsforge/vtm/blob/master/vtm-android-example/src/org/oscim/android/test/MapilionMvtActivity.java) example for how to add multiple _map layers_.

GroupLayer is more suitable for _simple_ overlays.
